AVAILABLE_LANGUAGES.MD

```diff
@@ -0,0 +1,169 @@
+## Available `language` props 
+* 1c
```

Comment:
we'll need to special case the handling of this one since var / export name can't start with a number